### PR TITLE
Observe worldtube position, velocity and acceleration

### PR DESCRIPTION
**Pull request by nikwit**
### Upgrade instructions

### CODE REVIEW

File: src/Evolution/Systems/CurvedScalarWave/Worldtube/SingletonActions/ObserveWorldtubeSolution.hpp [hunk 1]
```diff
@@ -78,18 +91,20 @@ struct ObserveWorldtubeSolution {
...
+          psi_coefs[1 + i + pos_offset] = psi_dipole.get(i);
+          psi_coefs[num_coefs + 1 + i + pos_offset] = dt_psi_dipole.get(i);
         }
       }
       const auto legend = [&expansion_order]() -> std::vector<std::string> {
         switch (expansion_order) {
           case (0):
-            return {"Time", "Psi0", "dtPsi0"};
+            return {"Time", "Posx", "Posy", "Posz", "Velx", "Vely",
+                    "Velz", "Accx", "Accy", "Accz", "Psi0", "dtPsi0"};
```
Comment (knelli2) [inline src/Evolution/Systems/CurvedScalarWave/Worldtube/SingletonActions/ObserveWorldtubeSolution.hpp]: To keep this consistent with our other output of 3-vectors, can you do `Position_{x,y,z}`, `Velocity_{x,y,z}`, and `Acceleration_{x,y,z}`?

**Review by knelli2 — CHANGES_REQUESTED** [review]
Squash immediately

**Comment by nikwit** [comment]
thanks for the review, I have squashed and rebased.

**Review by knelli2 — APPROVED** [review]

**Comment by knelli2** [comment]
Ignoring the clang-13 KerrSchild XCTS error because it's unrelated

**Pull request merged**